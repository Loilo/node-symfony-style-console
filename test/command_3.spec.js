

const wrapper = require('./wrapper')

test('03 Ensure has single blank line between two titles', wrapper(io => {
	io.title('First title')
	io.title('Second title')
}))
