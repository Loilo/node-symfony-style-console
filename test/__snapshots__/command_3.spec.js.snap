// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`03 Ensure has single blank line between two titles 1`] = `
"First title
===========

Second title
============"
`;
